### ISSUE TITLE
(marja): activate once parsing 'return' is merged into ParserBase. 

### ISSUE THREAD

**Issue by piascikj** [imdone/node#2442]
Issue created from code comment with imdone.io

----

TODO (marja): activate once parsing 'return' is merged into ParserBase. id:2441
[**deps/v8/test/cctest/test-parsing.cc:1501**](https://github.com/imdone/node/blob/9e8f4b3834263f0c66fb0f5483c4284a4d8ea682/deps/v8/test/cctest/test-parsing.cc#L1501)
